jp/components/drop_down.md

```diff
@@ -221,6 +221,56 @@ export class AppModule {}
     </button>
 </div>
 
+### igxDropDownItemNavigation ディレクティブ
+
+igxDropDownItemNavigation ディレクティブ
```

Comment:
@yoshimis I don't think this needs to be here?  Looks like a copy of the header without the `###`?